Makefile

```diff
@@ -73,26 +68,38 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: test
 test: manifests generate tools fmt vet ## Run tests.
 	$(shell go env GOPATH)/bin/staticcheck ./...
 	go install ./...
 	source <($(SETUP_ENVTEST) use -p env $(ENVTEST_K8S_VERSION)); \
 		go test -race -v -count 1 ./... --timeout=60s
 
+GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
+GOLANGCI_LINT_VERSION ?= v1.54.2
```

Comment:
This version setting should be in [versions.mk](https://github.com/topolvm/pvc-autoresizer/blob/main/versions.mk).

Reply:
The lint installation and version specification was changed in the following commit.
https://github.com/topolvm/pvc-autoresizer/pull/236/commits/e9077649d9ee8cb4c343ce1b64d0a139ca282075